corehq/apps/hqwebapp/tasks.py

```diff
@@ -93,6 +95,7 @@ def send_mail_async(self, subject, message, from_email, recipient_list,
             from_email=from_email,
```

Comment:
I just realized this will of course need to be part of the configuration

Reply:
yes it does, I added it here [e70be68](https://github.com/dimagi/commcare-hq/pull/33484/commits/e70be68dedee29ab617569ecf239f513145c16b3) and use it here for `send_mail_async`